ballerina/build.gradle

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2023 WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
Shall we use the 2024 license header in here? Check for other places as well

Reply:
Updated